idemp-base/src/org/bandahealth/idempiere/base/model/MBandaSetup.java

```diff
@@ -1359,6 +1360,40 @@ public boolean configureClientUsers() {
 		});
 		return true;
 	}
+	
+	/**
+	 * Create default business partners for new clients
+	 * @return
+	 */
+	public boolean createDefaultBusinessPartners() {
+		List<MBPartner_BH> patients = new Query(this.context, MBPartner_BH.Table_Name,
+				MBPartner_BH.COLUMNNAME_AD_Client_ID + "=?", getTransactionName()).setParameters(MClient_BH.CLIENTID_CONFIG)
+				.list();
+		patients.forEach((patient) -> {
+			if (!patient.save()) {
```

Comment:
One thing we'll need to add here is an update of any foreign keys to now point to the records saved for this client. As of right now, the business partner for this new client will point to a Group on the `config` client.

Reply:
Done